<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1181</width>
      <height>701</height>
     </rect>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
    <addaction name="actionSet"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>PE文件</string>
    </property>
    <widget class="QMenu" name="menu_PE">
     <property name="title">
      <string>可选PE数据目录</string>
     </property>
     <addaction name="actionResourceTable"/>
     <addaction name="actionExportTable"/>
     <addaction name="actionImportTable"/>
     <addaction name="actionBaseRelocationTable"/>
     <addaction name="actionBoundImportTable"/>
    </widget>
    <addaction name="actionParseDosHeader"/>
    <addaction name="actionParsePeHeader"/>
    <addaction name="actionParseOptionalHeader"/>
    <addaction name="actionParseSection"/>
    <addaction name="separator"/>
    <addaction name="menu_PE"/>
    <addaction name="separator"/>
    <addaction name="actionPeDetailParser"/>
   </widget>
   <widget class="QMenu" name="menuELF">
    <property name="title">
     <string>ELF文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionClearScreen"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
   <addaction name="menu_3"/>
   <addaction name="menuELF"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionClearScreen"/>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionParseDosHeader">
   <property name="text">
    <string>解析Dos头</string>
   </property>
  </action>
  <action name="actionParsePeHeader">
   <property name="text">
    <string>解析PE头</string>
   </property>
  </action>
  <action name="actionParseOptionalHeader">
   <property name="text">
    <string>解析可选PE头</string>
   </property>
  </action>
  <action name="actionSet">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="actionClearScreen">
   <property name="text">
    <string>清屏</string>
   </property>
  </action>
  <action name="actionParseSection">
   <property name="text">
    <string>解析区段表</string>
   </property>
  </action>
  <action name="actionPeDetailParser">
   <property name="text">
    <string>PE详细解析</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>可选PE数据目录</string>
   </property>
  </action>
  <action name="actionExportTable">
   <property name="text">
    <string>导出表</string>
   </property>
  </action>
  <action name="actionBaseRelocationTable">
   <property name="text">
    <string>重定位表</string>
   </property>
  </action>
  <action name="actionImportTable">
   <property name="text">
    <string>导入表</string>
   </property>
  </action>
  <action name="actionBoundImportTable">
   <property name="text">
    <string>绑定导入表</string>
   </property>
  </action>
  <action name="actionResourceTable">
   <property name="text">
    <string>资源表</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
